SR2019/2018-12-07-techday-announce.md

```diff
@@ -0,0 +1,22 @@
+---
+to: Student Robotics 2019
+subject: Announcing tech days
+---
+
+As was announced at kickstart, we intend to run some tech days over this competition year. Tech days are opportunities for teams to get help if they’re struggling with a technical issue, and is a controlled environment for students to make more progress on their robots. It’s also an opportunity to see how well they’re doing in comparison to other teams.
+
+As with all other events, students *must* be accompanied by a responsible adult.
+
+We will be running tech days at the University of Southampton on the following days:
+
+- 15th December 2018
+- 9th February 2019
+- 9th March 2019
```

Comment:
Perhaps include the day of week here, or if they're all the same then say something like "on the following Saturday:"?